qa/backwards-5.0/build.gradle

```diff
@@ -38,7 +38,7 @@ integTest {
   cluster {
     numNodes = 4
     numBwcNodes = 2
-    bwcVersion = "5.3.0-SNAPSHOT"
+    bwcVersion = "6.0.0-alpha1-SNAPSHOT" // TODO switch back to 5.3 and remove blacklist once backported
```

Comment:
I did it differently in another PR, I had my version checks initially .after with TODOs to make them onOrAfter once backported. That didn't cause any build failure.